modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/display/context/test/SegmentsExperienceSelectorDisplayContextTest.java

```diff
@@ -64,6 +65,8 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 
 		_layout = LayoutTestUtil.addTypeContentLayout(_group);
+
+		_selectedSegmentsExperienceId = -1;
```

Comment:
As you only use it in one method, better if you don't initialize this value in the setUp, it can cause side effects in other tests :(